can-stache-bindings.js

```diff
@@ -895,7 +895,8 @@ var startsWith = String.prototype.startsWith || function(text){
 	return this.indexOf(text) === 0;
 };
 var endsWith = String.prototype.endsWith || function(text){
-	return this.lastIndexOf(text) === (this.length - text.length);
+	var lastIndex = this.lastIndexOf(text);
+	return lastIndex !== -1 && lastIndex === (this.length - text.length);
```

Comment:
There are already tests written for this that fail in IE 9.

In Chrome `String.prototype.endsWith()` exists and works.

In combination with the changes to `test-saucelabs` we can make sure these tests run in IE 9 and fail.